Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Dockerfile # Plots2
 # https://github.com/publiclab/plots2
 
-FROM ruby:2.3
+FROM ruby:2.4.1
```

Comment:
I think you'll need to add "2.4.1-jessie" here as the container image is moving to Stretch (debian 9) by default. We did it in master branch like this: https://github.com/publiclab/plots2/blob/master/Dockerfile#L4.

Reply:
ok I will change that.